settings.py

```diff
@@ -2053,8 +2052,6 @@ def _determine_couch_databases(couch_databases):
     os.path.join('custom', 'enikshay', 'ucr', 'data_sources', 'qa', 'test.json'),
     os.path.join('custom', 'enikshay', 'ucr', 'data_sources', 'qa', 'voucher.json'),
 
-    os.path.join('custom', 'pnlppgi', 'resources', 'site_reporting_rates.json'),
-    os.path.join('custom', 'pnlppgi', 'resources', 'malaria.json'),
```

Comment:
@emord Do you know what's supposed to be done to remove static data sources?  I believe we can always rebuild the tables if needed, so we should be able to delete them, does that sound right?

Reply:
I need to do something like this for ICDS soon. If you just delete this from here, it won't actually delete the table from postgres. The table will just lie dormant on the hard drive forever.

Deleting is a bit messy because to delete it, you need to have the data source available in code, but if the data source is available then the pillow will rebuild it if it sees it's deleted. At this point you can manually drop the table, or once this is deployed in an old directory run something like:

```python
config, _ = get_datasource_config('id')
adapter = get_indicator_adapter(config)
config.drop_table()
```

It would be really cool if we had a management command that printed out old tables that we don't know anything about, and could automatically delete them (kind of like the elastic prune indices mgmt command).

Reply:
@emord Did you end up doing something along these lines?